config/data/18f.yaml

```diff
@@ -7,4 +7,8 @@ OFFICES:
   - 'Chicago'
   - 'Dayton'
   - 'San Francisco'
-  - 'Me! (Remote Worker)'
\ No newline at end of file
+  - 'Me! (Remote Worker)'
+RECURRENCE:
+  - 'Daily'
+  - 'Monthly'
+  - 'Yearly'
```

Comment:
This is okay for now, but I assume we'll want these values to have programatic meaning at some point, so we'll likely want them within the main code rather than in a configuration file.